src/voice-rss-api.js

```diff
@@ -171,6 +172,7 @@ class VoiceRSSWebApi {
             key: this.apiKey,
             src: text,
             hl: this.globalOptions.language ? this.globalOptions.language : (options.hl ? options.hl : 'en-us'),
+            v: this.globalOptions.voice ? this.globalOptions.voice : (options.v ? options.v : undefined),
```

Comment:
Is undefined an acceptable option for voice or is a different default needed?

Reply:
Yes, It is an acceptable option. When the voice parameter is left undefined, the API will use the default voice for the specified language.